### ISSUE TITLE
Packaging support

### ISSUE THREAD

**Issue by palfrey** [GrayJack/coreutils#79]
Is there any interest in PRs for packaging for various operating systems? e.g. Debian, Alpine, Homebrew for OS X (although they probably wouldn't accept it until there's a release)

**Comment by GrayJack** [comment]
There is, yes, I just don't do myself cause I dunno how to do it, the only one I know is for Arch based systems

**Comment by kegesch** [comment]
I could have a look homebrew packaging as soon as i have time.

**Comment by kegesch** [comment]
Also this might come in handy: [semantic-release](https://github.com/semantic-release/semantic-release)

**Comment by palfrey** [comment]
@kegesch semantic-release seems very Javascript-specific. Given the lack of a package.json in Rust projects, I'm not sure how well it'll work?

**Comment by kegesch** [comment]
@palfrey yes it is, but I don't think that it is limited to just js-projects. If it doesn't work with Rust, [this](https://github.com/semantic-rs/semantic-rs) should work though. It is just not really maintained anymore.
